plugins/X3DReader/X3DReader.py

```diff
@@ -11,71 +11,72 @@
 from UM.Math.Vector import Vector
 from UM.Mesh.MeshBuilder import MeshBuilder
 from UM.Mesh.MeshReader import MeshReader
-from UM.Scene.SceneNode import SceneNode
+#from UM.Scene.SceneNode import SceneNode
```

Comment:
Remove commented import line.

Reply:
Ah I forgot to check the total changes list before putting it into review... I'll clean this mess.